### PR TITLE
Added missing parameters and default value column

### LINKED ISSUE #3475: Parameter related issue on Deploy static view files
# Bug report
There is parameter are missing with on setup:static-content:deploy command

## Description
Check document https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-static-view.html

### PR DESCRIPTION
**Pull request by gelanivishal**
## This PR is a:

- [ ] New topic
- [X] Content update
- [ ] Content fix or rewrite
- [ ] Improvement

## Summary
I have added missing below parameters and added default value column for v2.x.
> --symlink-locale
> --content-version=CONTENT-VERSION

When this pull request is merged, it will fixed #3475 issue.

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.2/config-guide/cli/config-cli-subcommands-static-view.html
- https://devdocs.magento.com/guides/v2.1/config-guide/cli/config-cli-subcommands-static-view.html

### CODE REVIEW

File: guides/v2.1/config-guide/cli/config-cli-subcommands-static-view.md [hunk 1]
```diff
@@ -51,108 +51,401 @@ Command options:
...
-        <td>&lt;languages&gt;</td>
-        <td><p>Space-separated list of language codes to output static view files. (Default is <code>en_US</code>.)</p>
-        <p>You can find the list by running <code>magento info:language:list</code>.</p></td>
-    <td><p>No</p></td>
+  <col width="25%" />
+  <col width="60%" />
+  <col width="10%" />
+  <col width="10%" />
+  <tbody>
+    <tr>
```
Comment (osrecio) [inline guides/v2.1/config-guide/cli/config-cli-subcommands-static-view.md]: Can we have the `tr` and `th`  when is short `inline`:
e:g 
```
<tr>
	<th>Option</th>
	<th>Description</th>
	<th>Required?</th>
</tr>
```
We will increase the readbility. This comment should apply in all PR.

Please review it

Comment (gelanivishal) [reply guides/v2.1/config-guide/cli/config-cli-subcommands-static-view.md]: @osrecio I have updated table structure view. Please check it.

**Comment by jeff-matthews** [comment]
running tests

**Comment by jeff-matthews** [comment]
@gelanivishal, tests failed. Here's the error log:

```
Running ["LinkCheck"] on ["_site"] on *.html... 


Ran on 4819 files!


- internally linking to /guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-compact, which does not exist
  *  _site/guides/v2.1/config-guide/cli/config-cli-subcommands-static-view.html (line 3958)
     <a href="/guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-compact">compact strategy</a>
- internally linking to /guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-quick, which does not exist
  *  _site/guides/v2.1/config-guide/cli/config-cli-subcommands-static-view.html (line 3956)
     <a href="/guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-quick">quick strategy</a>
- internally linking to /guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-standard, which does not exist
  *  _site/guides/v2.1/config-guide/cli/config-cli-subcommands-static-view.html (line 3957)
     <a href="/guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-standard">standard strategy</a>
- trying to find hash of /guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-compact, but /apps/jenkins/latest/workspace/devdocs-test/_site/guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html does not exist
  *  _site/guides/v2.1/config-guide/cli/config-cli-subcommands-static-view.html (line 3958)
     <a href="/guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-compact">compact strategy</a>
- trying to find hash of /guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-quick, but /apps/jenkins/latest/workspace/devdocs-test/_site/guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html does not exist
  *  _site/guides/v2.1/config-guide/cli/config-cli-subcommands-static-view.html (line 3956)
     <a href="/guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-quick">quick strategy</a>
- trying to find hash of /guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-standard, but /apps/jenkins/latest/workspace/devdocs-test/_site/guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html does not exist
  *  _site/guides/v2.1/config-guide/cli/config-cli-subcommands-static-view.html (line 3957)
     <a href="/guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-standard">standard strategy</a>
rake aborted!
HTML-Proofer found 6 failures!
```

**Comment by gelanivishal** [comment]
@jeff-matthews I have added just content not an any link so How could I get this type of error?

**Comment by jeff-matthews** [comment]
@gelanivishal, the original files do not contain links to other topics in the table. Your changes introduce these links. See line 246 in the 2.1 file:

```
<tr>
         <td>--dry-run (-d)</td>	         <td>
         <td><p>Include to view the files output by the tool without outputting anything.</p></td>	             -s
         <td><p>No</p></td>	             <ul>
                 <li>-s quick</li>
                 <li>-s standard</li>
                 <li>-s compact</li>
             </ul>
         </td>
         <td>
             Define the deployment strategy. Use these options only if you have more than one locale.
             <ul>
                 <li>Use the <a href="{{ page.baseurl }}/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-quick">quick strategy</a> to minimize deployment time. This is the default command option if not specified.</li>
                 <li>Use the <a href="{{ page.baseurl }}/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-standard">standard strategy</a> to deploy all static view files for all packages.</li>
                 <li>Use the <a href="{{ page.baseurl }}/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-compact">compact strategy</a> to conserve disk space on the server.
                 </li>
             </ul>
         </td>
         <td>
             <p>No</p>
         </td>
     </tr>	     </tr>
```

**Comment by jeff-matthews** [comment]
I just ran a local test and did not receive any errors, which is strange. I'll try rerunning tests on this PR.

**Comment by jeff-matthews** [comment]
running tests

**Comment by jeff-matthews** [comment]
Yeah, this is still failing on Jenkins. I'll have to research it some more to find out why local tests pass but Jenkins tests fail.

**Comment by jeff-matthews** [comment]
@gelanivishal, after reviewing a little more closely, it's become clear that you added links in the 2.1 file to a topic that only exists in 2.2+. For example:

This
> https://devdocs.magento.com/guides/v2.2/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html

Does not exist in 2.1
>https://devdocs.magento.com/guides/v2.1/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by osrecio** [comment]
Hi @gelanivishal Can you check the comment from @jeff-matthews and update the PR.

Thanks you!

**Comment by osrecio** [comment]
@jeff-matthews can be closed this PR for inactivity?

**Comment by jeff-matthews** [comment]
Yes, good idea @osrecio.

**Comment by gelanivishal** [comment]
@jeff-matthews if file not exists in 2.1 so can I create that file for v2.1?

**Comment by jeff-matthews** [comment]
If the file doesn't exist in 2.1, then that probably means the information doesn't apply to the 2.1 version. So no, I don't think you can just create a 2.1 file. If a user is on version 2.1 and reading the docs, but they see a link functionality that only exists in 2.2, then that's not a good experience.

**Comment by gelanivishal** [comment]
@jeff-matthews But those links are already on that file. I haven't added those links. I have another content in it and You told me that you have to format code so. 
So, Can I remove those link in v2.1?

**Comment by jeff-matthews** [comment]
@gelanivishal, see the following comments:

- https://github.com/magento/devdocs/pull/3477#issuecomment-449410951
- https://github.com/magento/devdocs/pull/3477#issuecomment-449500104

**Pull request closed without merging**